Doc/library/time.rst

```diff
@@ -257,6 +257,13 @@ Functions
    :const:`None`, the current time as returned by :func:`.time` is used.  The dst
    flag is set to ``1`` when DST applies to the given time.
 
+   :func:`localtime` may raise :exc:`OverflowError`, if the timestamp is out of
+   the range of values supported by the platform C :c:func:`localtime` or
+   :c:func:`gmtime` functions, and :exc:`OSError` on :c:func:`localtime` or
+   :c:func:`gmtime` failure.
+   It's common for this to be restricted to years in
```

Comment:
```suggestion
   It's common for this to be restricted to years in the range of
```

Reply:
I just copied-pasted from `datetime`, but if we're to change the wording then there's some shorter options as well:

"... to be restricted to the years 1970 through 2038."

or

"... years in the range 1970 to 2038."

Though difficult for me to say which one of the three is the best here.